src/ParamConverter/DeveloperWithUser.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_edge\ParamConverter;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\ParamConverter\ParamConverterInterface;
+use Symfony\Component\Routing\Route;
+
+/**
+ * Resolves "developer_with_user" type parameters in routes.
+ *
+ * Use this parameter converter instead of paramconverter.entity if you would
+ * like to ensure that a developer (by UUID or email address) from Apigee Edge
```

Comment:
We should not be using UUIDs or supporting them, should only use email address in code and comments

Reply:
@cnovak I thought we have closed this topic with the two last comments in DRUP-489. We have to support them, but we can enforce usage of email address where it is possible.